src/ocm/services/InfraEnvsService.ts

```diff
@@ -23,6 +23,24 @@ const InfraEnvsService = {
     return infraEnvId;
   },
 
+  async getInfraEnv(clusterId: Cluster['id'], cpuArchitecture: CpuArchitecture) {
+    const infraEnvId = InfraEnvIdsCacheService.getInfraEnvId(clusterId, cpuArchitecture);
+    if (infraEnvId) {
+      return InfraEnvsAPI.get(infraEnvId);
+    } else {
+      throw new Error(
```

Comment:
Are you getting TS warnings without the else?
I think it shouldn't because it either throws an error or it returns the ID.

Reply:
I don't. I removed the else.

Reply:
Nevermind, I am getting TS warnings without the else. Just in a different file that I was expecting. The else is staying, I guess.